packages/autorest.typescript/test/rlcIntegration/lroRest.spec.ts

```diff
@@ -93,7 +93,7 @@ describe("LRO Rest Client", () => {
       if (isUnexpected(response)) {
         assert.fail(`Unexpected status code ${response.status}`);
       }
-      assert.equal(poller.getOperationState().status, "canceled");
+      assert.equal(poller?.operationState?.status, "canceled");
```

Comment:
https://github.com/Azure/autorest.typescript/pull/2443/files#diff-37bd95d53e94cd07f0b9e3056ff9ef0be5f33c55d49f1a3ddc62207d6f444751R115-R117 why we need to change the test since we have the interface here ?

Reply:
right, can we keep getOperationState() maybe we can just wrap opearationState. Also why are we nullchecking, does getLongRunningPoller return undefined?

Reply:
>  does getLongRunningPoller return undefined?
yes, because it is promise like now so the value would be undefined if the promise is not settled down.

Reply:
After adopting to V3 in RLC, we have two deprecated APIs which may be a breaking for customers. But considering the impact is limited I would prefer not taking them as breakings. The fact is 1) only 2 GAed RLC with LRO and 2) the toString is mainly used for rehydration which would be uncommon cases. The stopPolling is actually not supported in V2 version that's why we remove this in V3.

@qiaozha @joheredi How do you think?


No change:
- isDone, isStopped, getOperationState, getResult, poll, pollUntilDone, onProgress

Deprecated:
- toString, use `serialize` instead
- stopPolling, deprecated due to actually no support

Newly added:
- submitted
- serialize

Reply:
As discussed in the meeting, we would accept the `toString` breaking and remove this in `SimplePollerLike`. And we would investigate if this is feasible to support `stopPolling` in RLC.